library/Engine.nucleus.js

```diff
@@ -573,6 +573,7 @@ end
     if (!originUserID) throw new NucleusError.UndefinedValueNucleusError("The origin user ID must be defined.");
 
     const actionQueueName = await this.$actionDatastore.retrieveItemFromHashFieldByName(ACTION_QUEUE_NAME_BY_ACTION_NAME_ITEM_NAME_TABLE_NAME, actionName);
+    if (!nucleusValidator.isString(actionQueueName)) throw new NucleusError.UnexpectedValueTypeNucleusError(`The action queue name for ${actionName} doesn't exist.`);
```

Comment:
Can you change the message to something like `Could not executed the action "${actionName}" because it wasn't registered properly.`